backend/app/routes.py

```diff
@@ -323,6 +323,72 @@ async def get_game_status():
         return jsonify({
             "answer": answer
         }), 200
+        
+    @app.route('/more_explanation', methods=['POST'])
+    async def more_explanation():
+        data = await request.get_json()
```

Comment:
before data = await ... add this check:
`if not request.is_json:
            return jsonify({
                "type": "invalid_request",
                "message": "Request must be a JSON object."
            }), 400`

Reply:
between line 328 329